### PR TITLE
Jackson exclusion

### PR DESCRIPTION
**Pull request by shawnfeldman**

### CODE REVIEW

File: stack/rest/pom.xml [hunk 1]
```diff
@@ -408,6 +408,16 @@
...
+        <dependency>
+            <groupId>commons-httpclient</groupId>
+            <artifactId>commons-httpclient</artifactId>
+            <version>3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.jersey.contribs</groupId>
+            <artifactId>jersey-apache-client</artifactId>
+            <version>1.19</version>
+        </dependency>
```
Comment (shawnfeldman) [inline stack/rest/pom.xml]: needed because we are dependent on downstream dependencies from wns otherwise

File: stack/services/pom.xml [hunk 2]
```diff
@@ -434,16 +434,12 @@
...
           <groupId>com.github.fernandospr</groupId>
           <artifactId>java-wns</artifactId>
-          <version>1.3</version>
-          <exclusions>
-              <exclusion>
-                  <artifactId>jackson-jaxrs</artifactId>
-                  <groupId>org.codehaus.jackson</groupId>
-              </exclusion>
-          </exclusions>
+          <version>1.2-USERGRID</version>
```
Comment (shawnfeldman) [inline stack/services/pom.xml]: don't need exclusion anymore

File: stack/corepersistence/queue/src/main/java/org/apache/usergrid/persistence/queue/DefaultQueueManager.java [hunk 3]
```diff
@@ -18,33 +19,43 @@
...
+    import java.util.UUID;
+    import java.util.concurrent.ArrayBlockingQueue;
+    import java.util.concurrent.LinkedBlockingQueue;
+    import java.util.concurrent.PriorityBlockingQueue;
+    import java.util.concurrent.TimeUnit;
 
 /**
  * Default queue manager implementation, uses in memory linked queue
  */
 public class DefaultQueueManager implements QueueManager {
```
Comment (shawnfeldman) [inline stack/corepersistence/queue/src/main/java/org/apache/usergrid/persistence/queue/DefaultQueueManager.java]: our old impl was weak, this comes from master

**Comment by michaelarusso** [comment]
:+1:

**Pull request merged**